articles/accounts_certified_users_and_qualified_researchers.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: "Accounts, Certified Users and Qualified Researchers"
+title: "Synapse User Credentials including Certified Users and Qualified Researchers"
 layout: article
```

Comment:
We don't need to call out Synapse User Credentials as every article is about Synapse.

I suggest User Credentials including ...